toxcore/tox.c

```diff
@@ -55,7 +56,7 @@ struct Tox {
     // XXX: Messenger *must* be the first member, because toxav casts its
     // `Tox *` to `Messenger **`.
     Messenger *m;
-    Mono_Time *mono_time;
+    Mono_Time *mono_time; /* mono_time also needs to be at THIS position, if you put something before it things will crash!! */
```

Comment:
This is because of run_auto_test.h. Can you add that information to the comment? "Things" don't crash, only tests do because they rely on this fact.

Reply:
added comment